### ISSUE TITLE
umongo 0.7.5 does not provide the extra 'motor'

### ISSUE THREAD

**Issue by imbolc** [Scille/umongo#8]
```
pip install umongo[motor]
umongo 0.7.5 does not provide the extra 'motor'
```

**Comment by touilleMan** [comment]
v0.7.5 doesn't have patch fe155c6ad3002badbdea972201c9729373bcf9d2 https://github.com/Scille/umongo/blob/v0.7.5/setup.py

I release a v0.7.6 with it

**Comment by imbolc** [comment]
Sorry, still doesn't work:
```
(venv_motor)imbolc@ok:~/0/tests/test_umongo$ pip install umongo[motor]
Collecting umongo[motor]
  Downloading umongo-0.7.6-py2.py3-none-any.whl
Collecting marshmallow>=2.6.0 (from umongo[motor])
  Using cached marshmallow-2.7.2-py2.py3-none-any.whl
Collecting pymongo (from umongo[motor])
  Using cached pymongo-3.2.2.tar.gz
Collecting motor>=0.6.2 (from umongo[motor])
  Downloading motor-0.6.2.tar.gz (144kB)
    100% |████████████████████████████████| 147kB 637kB/s
Collecting greenlet>=0.4.0 (from motor>=0.6.2->umongo[motor])
  Downloading greenlet-0.4.9.zip (78kB)
    100% |████████████████████████████████| 81kB 625kB/s
Building wheels for collected packages: pymongo, motor, greenlet
  Running setup.py bdist_wheel for pymongo
  Stored in directory: /home/imbolc/.cache/pip/wheels/a6/d7/ea/bd149d9e2fd0f9ab21959b63c250ef509e4e38cc6258f6a07d
  Running setup.py bdist_wheel for motor
  Stored in directory: /home/imbolc/.cache/pip/wheels/2f/ee/d6/d674b432f15b0a2c3b3082954059f6d853ab5f6b84a4786c93
  Running setup.py bdist_wheel for greenlet
  Stored in directory: /home/imbolc/.cache/pip/wheels/54/85/ff/b9794e7f8dd8b845a7712885d2f7e8513d0e8fc1c91e01bc6f
Successfully built pymongo motor greenlet
Installing collected packages: marshmallow, pymongo, greenlet, motor, umongo
Successfully installed greenlet-0.4.9 marshmallow-2.7.2 motor-0.6.2 pymongo-3.2.2 umongo-0.7.6
```

**Comment by touilleMan** [comment]
It seems having pymongo both in `install_requires` and `extras_require` make pip only consider the first one...

So I've removed the dependency from `install_requires`. Installing by specifying which extra to use should be the prefered way now (i.g. `pip install umongo[pymongo]` instead of `pip install umongo`)

I've released a v0.7.7 with this ;-)
